packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -619,8 +650,12 @@ class MarkdownBuilder implements md.NodeVisitor {
     }
 
     Widget child;
-    if (imageBuilder != null) {
-      child = imageBuilder!(uri, title, alt);
+    if (imageBuilder != null || sizedImageBuilder != null) {
```

Comment:
Why are there lumped into one `if` branch and then subdivided by a ternary, given that there is no actual shared code in the two paths? It seems like this would be much clearer, and do less unnecessary work, as `if (sizedImageBuilder != null) {...} else if (imageBuilder != null) {...} else {...}`